css/base/form.css

```diff
@@ -0,0 +1,69 @@
+::placeholder, ::-ms-input-placeholder, :-ms-input-placeholder{
+  color: var(--placeholder);
+}
+
+.form-control, .form-control:focus{
+  width:324px;
+  height:45px;
+  border-radius: 73px;
+  background-color: var(--tph-tertiary);
+  color:#B9C8D3;
```

Comment:
There are also a lot of colors being used in the prototype that are not hard-coded within our css variables, such as this. Please identify their purpose and create :root variables for them with the appropriate name. Thanks!

Reply:
That is actually the hex value for input values, but it doesn't work on mine so I added an --input variable (to resolve this).